application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -86,13 +106,55 @@ export class ViewSearchInput
         this.input.actions.accept.subscribe(() => {
             const filter = this.input.asFilter();
             this.recent.update(filter.filter);
+            this.active = new ActiveSearch(this.session.search, filter);
+            this.active
+                .apply()
+                .then(() => {
+                    this.input.drop();
+                })
+                .catch((err: Error) => {
+                    this.log().error(`Fail to apply search: ${err.message}`);
+                })
+                .finally(() => {
+                    this.markChangesForCheck();
+                });
+        });
+        this.input.actions.flags.subscribe(() => {
+            if (this.active === undefined) {
+                return;
+            }
+            this.active
+                .setFlags(this.input.flags)
+                .apply()
+                .catch((err: Error) => {
+                    this.log().error(`Fail to set flags on search search: ${err.message}`);
+                })
+                .finally(() => {
+                    this.markChangesForCheck();
+                });
+        });
+
+        this.input.actions.accept.subscribe(() => {
```

Comment:
is this a mistake? You have 2 subscribers to `accept` (`this.input.actions.accept.subscribe`) on lines:
- 137
- 106

Reply:
Yes, I messed up on the merge conflict as it seems